sandbox/CMakeLists.txt

```diff
@@ -5,15 +5,12 @@ find_package(
   COMPONENTS Core Quick QuickControls2
   REQUIRED)
 
-add_executable(
-  ${PROJECT_NAME}
-  main.cpp
-  handler.cpp
-  sandboxapp.cpp
-  spellchecker.cpp
-  qml.qrc
-  ../src/statusq.qrc
-  ../src/assets.qrc)
+file(GLOB_RECURSE QML_FILES "../*.qml")
+
+qt5_add_big_resources(QRC_FILES qml.qrc ../src/statusq.qrc ../src/assets.qrc)
+
+add_executable(${PROJECT_NAME} main.cpp handler.cpp sandboxapp.cpp
+                               spellchecker.cpp ${QRC_FILES} ${QML_FILES})
```

Comment:
Here you don't need to add the `QML_FILES` to the binary, they are already in `${QRC_FILES}`

Reply:
Right... you could create a dummy target for that:
```
# IDE support: Add dummy target so one can jump to .qml files (needed for QtCreator at least)
file(GLOB_RECURSE ALL_QML_FILES "*.qml")
add_library(dummy_target_with_all_qml_files EXCLUDE_FROM_ALL ${ALL_QML_FILES})
```

Reply:
Nice trick but I don't see any benefit of having it that way. IMO adding qml files to the executable describes intention better -"this executable depends on these files". Also, dummy targets will only obscure IDEs' explorer views (e.g. QtCreator).